app/helpers/events_helper.rb

```diff
@@ -1,10 +1,18 @@
 module EventsHelper
   def next_event
-    @next_event ||= Eventbrite::EventFinder.find(ENV.fetch("NEXT_EVENT_ID"))
+    OpenStruct.new(
+      venue: ENV.fetch("VENUE_MAP_URL"),
+      url: ENV.fetch("EVENT_URL")
+    )
+    # @next_event ||= Eventbrite::EventFinder.find(ENV.fetch("NEXT_EVENT_ID"))
   end
 
   def next_event_venue
-    next_event.venue
+    OpenStruct.new(
+      name: ENV.fetch("VENUE_NAME"),
+      address: ENV.fetch("VENUE_ADDRESS")
+    )
+    # @next_event.venue
```

Comment:
Can you comment out the original version (i.e. without the `@`) so if we want to return to this, it will be exactly the same?